### DISCUSSION TITLE
Deploying to Digital Ocean App platform, what happenes to images stored?

### DISCUSSION THREAD

**Discussion by NafunaAfrica** [Vontigo/Vontigo#105] [General]
Wanted to know id i use Vontigo on my existing sveltekit site which is on digital ocean app platform, whre are the images in posts and blogs stored? In the sql lite database? What happens when I then update the codebase for the whole site? the content would be lost when the app auto pulls the repo and github and builds the front end

**Comment by Smilefounder** [comment]
Hello @NafunaAfrica 

Thank you for reaching out and expressing your interest in getting involved with our SvelteKit project. It's great to hear that you find our project exciting, and we welcome your enthusiasm to contribute.

Regarding your question about hosting your current app on Digital Ocean App Platform and what happens when the platform rebuilds the frontend after a code update:

When you update your code and Digital Ocean's App Platform rebuilds your frontend, the content you have uploaded, such as images and posts, should not be lost by default. Digital Ocean's App Platform generally separates your application code from your data, so code updates should not affect your data.

Best regards,
Huy Nguyen
